campaigns/zcp/mourning_chorus.json

```diff
@@ -2005,10 +2005,277 @@
         }
       ]
     },
+    {
+      "id": "record_rallied_version",
+      "hidden": true,
+      "type": "branch",
+      "condition": {
+        "type": "campaign_data",
+        "campaign_data": "version",
+        "min_version": 4,
+        "options": [
+          {
+            "boolCondition": true,
+            "steps": [
+              "record_rallied"
+            ]
+          },
+          {
+            "boolCondition": false,
+            "steps": [
+              "record_rallied_legacy"
```

Comment:
So the old one should keep the name "record_rallied" while the new one will need the new name ("record_rallied_new") -- basically we want to maintain the ID for the old version on the input.